LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -1032,26 +1035,30 @@ BriefSummary="Увеличивает меткость на <Ability:SCOPE_SUP_AI
 TinySummary="Повышенная меткость при обычных выстрелах"
 LootTooltip="Это улучшение повысит меткость любого оружия, на которое оно будет установлено, при обычной (не ответной) стрельбе."
 
+
+LWOTC Translated
 [MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
-FriendlyName="Приклад"
-FriendlyNamePlural="Приклады"
-BriefSummary="Дает способность \"Устойчивая позиция\", добавляющую +<Ability:STOCK_BSC_SW_AIM_BONUS/> к меткости и +<Ability:STOCK_BSC_SW_AIM_BONUS/> к вероятности критического попадания при следующем выстреле."
-TinySummary="Дает способность \"Устойчивая позиция\""
-LootTooltip="Это улучшение дает солдату способность \"Устойчивая позиция\"."
+FriendlyName="Рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
 
 [MissDamageUpgrade_Adv X2WeaponUpgradeTemplate]
-FriendlyName="Модифицированный приклад"
-FriendlyNamePlural="Модифицированные приклады"
-BriefSummary="Дает способность \"Устойчивая позиция\", дающую +<Ability:STOCK_ADV_SW_AIM_BONUS/> к меткости и +<Ability:STOCK_ADV_SW_AIM_BONUS/> к вероятности крит. попадания при след. выстреле."
-TinySummary="Дает способность \"Устойчивая позиция\""
-LootTooltip="Это улучшение дает солдату способность \"Устойчивая позиция\"."
+FriendlyName="Улучшенная рукоятка"
+FriendlyNamePlural="Улучшенные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
```

Comment:
As above, please use the ability tag, and for the superior version below.

Reply:
Done